benches/Cargo.toml

```diff
@@ -28,7 +31,6 @@ name = "witness"
 path = "bitcoin/witness.rs"
 harness = false
 
-
```

Comment:
Whitespace change shouldn't really be in this PR.

Reply:
Reverted all the whitespace changes in `Cargo.toml`s.

Reply:
Typically we don't like whitspace changes (or any unrelated change) in a patch because it makes the review require more brain power. So it costs the reviewer more and the dev less, we want to invert this since review time is more valuable than dev time (because there is less of it).

Thanks bro